uint8[] joint_status